src/instructions/timed_exam/SubmitTimedExamInstructions.jsx

```diff
@@ -27,19 +28,20 @@ const SubmitTimedExamInstructions = () => {
           defaultMessage="After you submit your exam, your exam will be graded."
         />
       </p>
-      <Button variant="primary" onClick={submitExam}>
+      <Button variant="primary" onClick={submitExam} className="mr-2" data-testid="end-exam-button">
         <FormattedMessage
           id="exam.submitExamInstructions.submit"
           defaultMessage="Yes, submit my timed exam."
         />
       </Button>
-      &nbsp;
```

Comment:
ditto

Reply:
answered above

Reply:
Got it thanks! I think we can stay with the current approach